bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -342,18 +342,40 @@ async def end_nomination_command(self, ctx: Context, user: MemberOrUser, *, reas
             await ctx.send(":x: The specified user does not have an active nomination")
 
     @nomination_group.group(name='edit', aliases=('e',), invoke_without_command=True)
-    @has_any_role(*MODERATION_ROLES)
-    async def nomination_edit_group(self, ctx: Context) -> None:
-        """Commands to edit nominations."""
-        await ctx.send_help(ctx.command)
+    @has_any_role(*STAFF_ROLES)
+    async def nomination_edit_group(self, ctx: Context, target: Member, *, reason: str) -> None:
+        """
+        Commands to edit nominations.
+
+        Edits the reason of the authors nominee in a specific target's active nomination.
+        """
+        await self._edit_nomination_reason(ctx, target=target, actor=ctx.author, reason=reason)
 
     @nomination_edit_group.command(name='reason')
     @has_any_role(*MODERATION_ROLES)
     async def edit_reason_command(self, ctx: Context, nomination_id: int, actor: MemberOrUser, *, reason: str) -> None:
         """Edits the reason of a specific nominator in a specific active nomination."""
+        await self._edit_nomination_reason(ctx, target=nomination_id, actor=actor, reason=reason)
+
+    async def _edit_nomination_reason(
+        self,
+        ctx: Context,
+        target: Union[int, Member],
+        actor: MemberOrUser,
+        reason: str,
+    ) -> None:
+        """Edits a nomination reason in the database after validating the input."""
         if len(reason) > REASON_MAX_CHARS:
-            await ctx.send(f":x: Maxiumum allowed characters for the reason is {REASON_MAX_CHARS}.")
+            await ctx.send(f":x: Maximum allowed characters for the reason is {REASON_MAX_CHARS}.")
             return
+        if isinstance(target, Member):
+            if nomination := self.cache.get(target.id):
+                nomination_id = nomination["id"]
+            else:
+                await ctx.send("No active nomination found for that member.")
+                return
+        else:
+            nomination_id = target
```

Comment:
How does this work when multiple staff members nominate the same user?

Reply:
The `actor` kwarg is what controls which nomination to edit.

IE If you nominated Akarys, I would set the target to Akarys and the actor to you to update your nomination.

This is how we ensure helpers can only edit their own nominations, since we hardcode actor to be ctx.author in the helper command

Reply:
But the lookup is done by target ID. Do nominations for the same user have the same ID?

Reply:
Yea, that's right. Each nomination ID contains a list of nomination entries from different actors

Reply:
Hmm wait, I don't get it.

- Each helper can have one nomination of someone
- A helper can only edit their own nomination of that person

So that means that a nomination is identifiable by the actor and the nominee.
So that removes the need for an ID.

Reply:
Yea, helpers don't need to supply an ID when running !tp edit, since the ID is retrieved on lines 372-374.

My comments above were in an attempt to describe the structure